dashboard/lib/tasks/seed.rake

```diff
@@ -150,6 +150,69 @@ namespace :seed do
     oceans
     sports
   ).map {|script| "config/scripts_json/#{script}.script_json"}.freeze
+
+  ADHOC_TEST_SCRIPTS = UI_TEST_SCRIPTS + %w(
+    allthehiddenthings
+    allthemigratedthings
+    alltheplcthings
+    alltheselfpacedplthings
+    allthethings
+    allthettsthings
+    coursea-2024
+    courseb-2024
+    coursec-2024
+    coursed-2024
+    coursee-2024
+    coursef-2024
+    csd1-2024
+    csd2-2024
+    csd3-2024
+    csd4-2024
+    csd5-2024
+    csd6a-2024
+    csd6b-2024
+    csd7-2024
+    csd-post-survey-2024
+    interactive-games-animations-2024
+    focus-on-creativity3-2024
+    focus-on-coding3-2024
+    csp1-2024
+    csp2-2024
+    csp3-2024
+    csp4-2024
+    csp5-2024
+    csp6-2024
+    csp7-2024
+    csp8-2024
+    csp9-2024
+    csp10-2024
+    csp-post-survey-2024
+    csa1-2024
+    csa2-2024
+    csa3-2024
+    csa4-2024
+    csa5-2024
+    csa6-2024
+    csa7-2024
+    csa8-2024
+    csa9-2024
+    csa-postap-se-and-computer-vision-2024
```

Comment:
I think it is important to emphasize that in 2025 we should not add to this list, rather the 2024 courses should be replaced with their 2025 versions (if there is a newer version).My recommendation would be to extract these 2024 units into a new array named MOST_RECENT_ADHOC_COURSES or similar, with a comment which indicates how this list should be updated.